crates/wash-cli/src/bin/wash.rs

```diff
@@ -264,8 +275,17 @@ struct Cli {
     )]
     help_markdown: bool,
 
+    #[clap(
+        short = 'V',
+        long = "version",
+        help = "Print version",
+        default_value = "false",
```

Comment:
supernit: Isn't the default value false here already?

Reply:
True. I have removed the redundant line. Thanks.